README.md

```diff
@@ -1907,14 +1907,34 @@ $ unset GNUPGHOME
 
 # Using keys
 
+The following command creates and initialises the `~/.gnupg` if it does not exist already:
+
+```console
+$ gpg -k
+```
+
+Change your working directory:
+
+```console
+$ cd ~/.gnupg
+```
+
 Download [drduh/config/gpg.conf](https://github.com/drduh/config/blob/master/gpg.conf):
 
 ```console
-$ cd ~/.gnupg ; wget https://raw.githubusercontent.com/drduh/config/master/gpg.conf
+$ wget https://raw.githubusercontent.com/drduh/config/master/gpg.conf
 
 $ chmod 600 gpg.conf
 ```
 
+Set the following option. This avoids the problem where GnuPG will prompt, repeatedly, for the insertion of an already-inserted YubiKey:
+
+```console
+$ echo "disable-ccid" >>~/scdaemon.conf
```

Comment:
This should be `~/.gnupg/scdaemon.conf` or just `scdaemon.conf`, right?

Reply:
Yes. Blush.

Reply:
Updated pushed.